datasets/ccaligned_multilingual/ccaligned_multilingual.py

```diff
@@ -0,0 +1,213 @@
+"""
+
+_HOMEPAGE = "http://www.statmt.org/cc-aligned/"
+
+
+_LICENSE = ""  # Unknown
+
+
+_URLs = {
+    "documents": "http://www.statmt.org/cc-aligned/",
+    "sentences": "http://www.statmt.org/cc-aligned/sentence-aligned/",
+}
+
+reverse_mapped_sentences = [
+    "af_ZA",
+    "ak_GH",
+    "am_ET",
+    "ar_AR",
+    "as_IN",
+    "ay_BO",
+    "az_AZ",
+    "az_IR",
+    "be_BY",
+    "bg_BG",
+    "bm_ML",
+    "bn_IN",
+    "br_FR",
+    "bs_BA",
+    "ca_ES",
+    "cb_IQ",
+    "cs_CZ",
+    "cx_PH",
+    "cy_GB",
+    "da_DK",
+    "de_DE",
+    "el_GR",
+]  # Some languages have the reverse source languages in the URLs.
+
+
+class CCAlignedMultilingualConfig(datasets.BuilderConfig):
+    def __init__(self, *args, type=None, language_code=None, **kwargs):
+        super().__init__(
+            *args,
+            name=f"{type}-{language_code}",
+            **kwargs,
+        )
+        self.type = type
+        self.language_code = language_code
+
+
+class CCAlignedMultilingual(datasets.GeneratorBasedBuilder):
+    """The CCAligned Multilingual Dataset."""
+
+    VERSION = datasets.Version("1.0.0")
+
+    BUILDER_CONFIGS = [
+        CCAlignedMultilingualConfig(
+            type="documents",
+            language_code="zz_TR",
+            version=VERSION,
+            description="The dataset containing document-pairs for en_XX-zz_TR.",
+        ),
+        CCAlignedMultilingualConfig(
+            type="sentences",
+            language_code="zz_TR",
+            version=VERSION,
+            description="The dataset containing sentence-pairs for en_XX-zz_TR.",
+        ),
+        CCAlignedMultilingualConfig(
+            type="documents",
+            language_code="tz_MA",
+            version=VERSION,
+            description="The dataset containing document-pairs for en_XX-tz_MA.",
+        ),
+        CCAlignedMultilingualConfig(
+            type="sentences",
+            language_code="tz_MA",
+            version=VERSION,
+            description="The dataset containing sentence-pairs for en_XX-tz_MA.",
+        ),
+        CCAlignedMultilingualConfig(
+            type="documents",
+            language_code="ak_GH",
+            version=VERSION,
+            description="The dataset containing document-pairs for en_XX-ak_GH.",
+        ),
+        CCAlignedMultilingualConfig(
+            type="sentences",
+            language_code="ak_GH",
+            version=VERSION,
+            description="The dataset containing sentence-pairs for en_XX-ak_GH.",
+        ),
+    ]
+
+    BUILDER_CONFIG_CLASS = CCAlignedMultilingualConfig
+
+    DEFAULT_CONFIG_NAME = "documents-zz_TR"
+
+    def _info(self):
+        if self.config.name[:9] == "documents":
+            features = datasets.Features(
+                {
+                    "Domain": datasets.Value("string"),
+                    "Source_URL": datasets.Value("string"),
+                    "Source_Content": datasets.Value("string"),
```

Comment:
Can you use the Translation feature type here please ?
```
"translation": datasets.Translation(languages=("en_XX", self.config.language_code))
```
You'll need to regenerate the dataset_infos.json file after this change